examples/template-react-ts/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
Should we ditch the favicons and the CSS? Might mean a bit more noise in examples

Reply:
We should probably replace the favicons with XState favicons 🤔 will do that in a follow-up